front50-gcs/src/main/java/com/netflix/spinnaker/front50/model/GcsStorageService.java

```diff
@@ -206,6 +206,14 @@ public T call() throws IOException {
   }
 
   /**
+   * Create a modification file if needed to avoid logging errors later.
+   */
+  public void initObjectType(ObjectType objectType) {
+      log.info("Initializing {}", objectType.group);
+      getLastModifiedHelper(objectType, false);
```

Comment:
Seems like an oddly named method to me ... with an unclear side-effect that a last modified file will be created.